tests/test_timereport.py

```diff
@@ -23,7 +23,7 @@ def test_slack_payload_extractor():
 
 
 def test_factory():
-    fake_order = dict(user_id='fake', user_name='fake mcFake', text='fake_cmd=do_fake+fake_reason+2018-01-01')
+    fake_order = dict(user_id='fake', user_name='fake mcFake', text='fake_cmd=do_fake fake_reason 2018-01-01')
     fake_result = factory(fake_order)
```

Comment:
The request passed by apigw from slack contains "+" instead of spaces. Changing to spaces will make the test less like real world scenario